test/conformance/testdata/conformance.yaml

```diff
@@ -330,6 +330,21 @@
     resources are available.
   release: v1.16
   file: test/e2e/apimachinery/custom_resource_definition.go
+- testname: custom-resource-definition-field-selectors-list-watch-register-informers
+  codename: '[sig-api-machinery] CustomResourceFieldSelectors [Privileged:ClusterAdmin]
+    [FeatureGate:CustomResourceFieldSelectors] [Beta] CustomResourceFieldSelectors
```

Comment:
I guess manually remove L335-336? I'm able to replicate the diff pulling your branch and running `update-conformance-yaml.sh`

Reply:
![image](https://github.com/user-attachments/assets/33986617-343a-459e-8d30-1cdc6fbd2e3b)